cargo-concordium/src/main.rs

```diff
@@ -346,10 +347,12 @@ pub fn main() -> anyhow::Result<()> {
                     ref context,
                     ..
                 } => {
-                    let init_ctx: InitContext = {
-                        let ctx_file = fs::read(context).context("Could not open context file.")?;
-                        serde_json::from_slice(&ctx_file)
-                            .context("Could not parse the init context JSON.")?
+                    let init_ctx: InitContextOpt = {
+                        match fs::read(context) {
+                            Ok(ctx_file) => serde_json::from_slice(&ctx_file)
+                                .context("Could not parse the init context JSON.")?,
+                            Err(_) => InitContextOpt::new(),
```

Comment:
I don't think this is a good idea. If we fail reading the supplied file then something is not right, and we should not just silently ignore it. We should just terminate the program with the error message from `fs::read`.

We could, on the other hand, make the `context` parameter optional, and if not present then use `InitContextOpt::new`.